index.d.ts

```diff
@@ -0,0 +1,18 @@
+export function createContext(): Record<any, any>
```

Comment:
```suggestion
export function createContext(): object
```

Reply:
Not sure if it's a problem but I usually compile the typings together with our test files to see the impacts in the wild, using `object` I got:

```
test.js:9:11 - error TS2339: Property 'x' does not exist on type 'object'.

9   context.x = 42
```

Reply:
Good point, the context isn't just an opaque object. Let's make it a `Record<unknown, unknown>`.

Reply:
`Record<unknown, unknown>` unlocked a new issue:

```
index.d.ts:1:41 - error TS2344: Type 'unknown' does not satisfy the constraint 'string | number | symbol'.
1 export function createContext(): Record<unknown, unknown>
                                          ~~~~~~~

index.d.ts:5:19 - error TS2344: Type 'unknown' does not satisfy the constraint 'string | number | symbol'.
5   context: Record<unknown, unknown>,
```

I've changed then to `Record<string | number | symbol, unknown>`, it does seem to do the trick.